src/pages/_app.tsx

```diff
@@ -1,93 +1,91 @@
-import App, { AppProps } from "next/app";
+import { AppProps } from "next/app";
 import Head from "next/head";
 import { useEffect, useState } from "react";
 import { QueryClient, QueryClientProvider } from "react-query";
 
-import { CookieConsent } from "@/components/cookies/CookieConsent";
 import ErrorBoundary from "@/components/error/ErrorBoundary";
+import { Overlays } from "@/components/organisms/overlays/Overlays";
+import { COOKIE_FEATURES_NAME } from "@/constants/cookies";
+import { DEFAULT_THEME_CONFIG } from "@/constants/themeConfig";
 import { AdobeContext } from "@/context/AdobeContext";
-import { EnvConfigContext } from "@/context/EnvConfig";
+import { NewFeatureContext } from "@/context/NewFeatureContext";
 import { PostHogProvider } from "@/context/PostHogProvider";
 import { ThemeContext } from "@/context/ThemeContext";
-
 import "../styles/flag-icons.css";
 import "../styles/main.css";
+import { TTheme, TThemeConfig } from "@/types";
+import { getCookie, setCookie } from "@/utils/cookies";
+import getDomain from "@/utils/getDomain";
+import { readConfigFile } from "@/utils/readConfigFile";
+
+const theme = (process.env.THEME ?? "cpr") as TTheme;
+const adobeApiKey = process.env.ADOBE_API_KEY ?? "";
 
 const favicon = `/images/favicon/${process.env.THEME}.png`;
 
 const queryClient = new QueryClient();
 
-interface IProps extends AppProps {
-  theme?: string;
-  adobeApiKey?: string;
-}
+function MyApp({ Component, pageProps }: AppProps) {
+  const [previousNewFeature, setPreviousNewFeature] = useState<number | null>(null);
+  const [displayNewFeature, setDisplayNewFeature] = useState<number | null>(null);
+  const [themeConfig, setThemeConfig] = useState<TThemeConfig>(DEFAULT_THEME_CONFIG);
 
-function MyApp({ Component, pageProps, theme, adobeApiKey }: IProps) {
-  const [siteTheme, setSiteTheme] = useState(null);
-  const [adobeKey, setAdobeKey] = useState(null);
+  const getThemeConfig = async () => {
```

Comment:
Question: because we are getting this and then updating the context _after_ the app has loaded. Are we creating a risk there will be an instance where the config is required but won't be loaded in time? (race condition?)